tools/devtools/nodes/inputs.py

```
@@ -445,6 +445,31 @@ async def execute(cls):
         return IO.NodeOutput()
 
 
+class NodeWithComboControlWidget:
+    @classmethod
+    def INPUT_TYPES(cls):
```

Ruff (0.16.3)

[warning] Missing return type annotation for classmethod `INPUT_TYPES`

(ANN206)